src/components/exercise-identifier-link.cjsx

```diff
@@ -12,11 +12,20 @@ ExerciseIdentifierLink = React.createClass
     exerciseId: React.PropTypes.string.isRequired
     project: React.PropTypes.oneOf(['concept-coach', 'tutor'])
 
+  renderSection: ->
+    section = _.first(@props.related_content)
+    if (@props.project isnt 'concept-coach' or not section) then return
+
+    <span className="related-section">
+      Comes from {@sectionFormat(section.chapter_section)} - {section.title}
+    </span>
+
   render: ->
```

Comment:
I don't think we want to add the "Comes from ..." to the `ExerciseIdentifierLink`.  Since HW  also use the `IdentifierLink` and also include the message themselves (added by @darrenmason earlier), they'll have the message appear twice.  

See:

<img width="992" alt="screen shot 2016-07-29 at 4 42 45 pm" src="https://cloud.githubusercontent.com/assets/79566/17262812/a6a2cda0-55ab-11e6-9ee8-a948bb8d9bb0.png">